.circleci/config.yml

```diff
@@ -670,10 +670,14 @@ jobs:
       - install-contracts-dependencies
       - check-changed:
           patterns: contracts-bedrock
-      - run:
-          name: Check frozen files
-          command: just check-frozen-code
-          working_directory: packages/contracts-bedrock
+      - when:
+          condition:
+            equal: ['develop', << pipeline.git.branch >>]
```

Comment:
Does this mean it only runs on develop, or when the target branch is develop? https://circleci.com/docs/pipeline-variables/ says "The name of the git branch that was pushed to trigger the pipeline" which feels like this would now run only after merge to develop?

Reply:
Pushed f14c34f which should resolve this